src/modalities/dataloader/preprocessing/chunking/create_chunks.py

```diff
@@ -47,5 +47,6 @@ def get_file_chunk(dataset: PackedMemMapDatasetBase, num_chunks: int, chunk_id:
         return chunk
 
     @staticmethod
-    def shuffle_file_chunks_in_place(file_chunks: list[np.ndarray]):
-        np.random.shuffle(file_chunks)
+    def shuffle_file_chunks_in_place(file_chunks: list[np.ndarray], seed: int = None) -> None:
```

Comment:
```suggestion
    def shuffle_file_chunks_in_place(file_chunks: list[np.ndarray], seed: Optional[int] = None) -> None:
```
would require `from typing import Optional`.